tests/pages/components/user-sidebar.js

```diff
@@ -0,0 +1,13 @@
+import { count, text, isHidden } from 'ember-cli-page-object';
+
+export default {
+  scope : '.user-sidebar',
```

Comment:
could it be because `count('.user-sidebar')` will look for `.user-sidebar` _within_ the scope (which is `.user-sidebar`)?